VOER-1817: Added support for testing on aarch64.

safiyat opened the pull request:
Ported asm cpuid to arm64.
asm cpuid needs porting to arm64.
Porting to arm64.
Changing uint8_t to uint32_t for RTE_MAX_LCORE becasue if RTE_MAX_LCORE >= 256, uint8_t cause infinite loop as it values between 0 to 255

deebhatia commented on VNFs/DPPD-PROX/prefetch.h:
```diff
@@ -21,7 +21,9 @@
 
 static inline void prefetch_nta(volatile void *p)
 {
+	#if defined(__x86_64__)
 	asm volatile ("prefetchnta %[p]" : [p] "+m" (*(volatile char *)p));
+	#endif
```
add a comment why we have not added any ARM alternative here.

safiyat replied:
Added.

deebhatia reviewed (commented):
Also raise the PR upstream on latest master: https://gerrit.opnfv.org/gerrit/q/project:samplevnf

safiyat committed:
VOER-1817: Added support for testing on aarch64.

Ported asm cpuid to arm64.
asm cpuid needs porting to arm64.
Porting to arm64.
Changing uint8_t to uint32_t for RTE_MAX_LCORE becasue if RTE_MAX_LCORE >= 256, uint8_t cause infinite loop as it values between 0 to 255

Signed-off-by: Md Safiyat Reza <safiyat@voereir.com>
